### DISCUSSION TITLE
Statuskanal(HmIP-Broll Channel3) aktivieren?

### DISCUSSION THREAD

**Discussion by Lemocin** [SukramJ/aiohomematic#39] [Ideas]
Hi, 
in der aktuellen Version ist der HmIP-BROLL als Cover integriert. 
Was mir allerdings gut in den vorherigen Versionen gefallen hat, ist, das man z.B. den Kanal 3 mit der Statusmitteilung hatte, den man in einem Template Cover benutzen konnte. Nun ist es wieder so, dass bei einer manuellen Betätigung des Rollladens mit zwischenzeitlichem Stopp bei z.B. 50% immer 0 oder 100 angezeigt werden. 
Wäre es eine Option, besonders bei Nichtbenutzung der virtuellen Kanäle, den Statuskanal weiter zu verwenden oder optional zu aktivieren? Auch der Sensor wie die Temperatur vom Schalter war interessant zu sehen. 
Oder gibt es vielleicht schon so eine Möglichkeit?
Danke für die Klasse Arbeit, die Integration gefällt mir sehr!

**Comment by danielperna84** [comment]
Also der Plan wäre eigentlich, dass der Status vom Cover-entity den Wert hat, den du vorher im Sensor hattest. Das ist ja mit einer der Hauptgründe für die neue Integration. Insofern ist es vermutlich ein sinnvolles Feedback, dass das wohl (noch) nicht funktioniert.

**Reply by Lemocin** [reply]
Hi Daniel,
danke für die Info. 
Mir ist aufgefallen, dass der "reale" Wert in den Attributen existiert, solange er nicht mit dem Schieberegler übereinstimmt. Also er scheint schonmal da zu sein, nur halt nicht als Status sondern als Attribut. 
![grafik](https://user-images.githubusercontent.com/4162496/141793181-22ce7299-98b4-4517-9595-9e8bfd265999.png)

**Comment by SukramJ** [comment]
Der aktuelle Stand soll erst mal zeigen das Cover überhaupt funktioniert, da ich selber keine Geräte dazu besitze.
Also funktioniert rauf/runter/stop?

Wenn Cover grundsätzlich funktioniert, würde ich den Status über Ch:3 (Sensor) anzeigen und die Steuerung über Ch:4.
Des weitern würde ich dann die virtuellen Kanäle (Ch:5+6) deaktivieren, da die dann keine Sinn mehr machen.

**Reply by Lemocin** [reply]
Rauf/Runter/Stopp und Lage funktionieren. Channel 5 und 6 sind auch weg, sobald ich die Checkbox mit den virtuellen Channels aus habe.

**Reply by SukramJ** [reply]
Oben ist auch oben und unten ist unten?

**Reply by Lemocin** [reply]
Jop, Die Richtung stimmt

**Comment by SukramJ** [comment]
@Lemocin Kannst Du mal die aktuelle Version mit deinem Broll testen? Der Status wird jetzt von Ch3 übernommen.

**Reply by Lemocin** [reply]
Ja, seit ca einer Woche stimmt das alles schon. Auch in der Version von gestern. Nur operating Voltage ist unbekannt. Die habe ich einfach deaktiviert.

Danke!

**Reply by SukramJ** [reply]
Danke für die Rückmeldung.

In Bezug auf Operating Voltage und auch Rssi Peer:
Es gibt ein paar Parameter im ParamSet Ch0 die haben bei einigen Devices Werte und bei anderen nicht.
Ich hab mal ein wenig rumgespielt, und die Entitäten deaktiviert, wenn dazu keine Werte geladen werden konnten.
Letztlich hab ich den Teil wieder aus kommentiert, weil mir nicht klar war, ob bei allen Entitäten zu dem Zeitpunkt Werte geladen werden konnten. Mein HmIP-SLO liefert z. Bsp erst sehr spät Werte.
Einfach in der UI deaktivieren ist, denke ich, der richtige Weg.

**Comment by kwithus** [comment]
Wenn die Rollladen fährt, wird in Lovelace ‚öffnet’ oder ‘schließt’ angezeigt, was natürlich schon super ist. Wenn er dann aber zwischendrin stoppt, wird auch bei z.B. 50% offen angezeigt. Wenn man auf die card klickt sieht man, dass der Schieber bei 50% steht. Kann man das so einstellen, dass der Zustand (Chan 3) auch in der Lovelace Card direkt angezeigt wird?

**Reply by SukramJ** [reply]
HA Logik:
Alles was nicht geschlossen ist ist offen.

**Reply by SukramJ** [reply]
Der Ch3 wird als Status für den BROLL verwendet.

**Reply by kwithus** [reply]
Ok, das heißt, dass HA standardmäßig in der Lovelace Card nicht den Status, sondern nur offen oder geschlossen anzeigt (anzeigen kann) und der Status zwar richtig in der entity drin ist, aber eben nicht dort angezeigt werden kann?
Sorry für die Nachfrage, aber auch wenn ich mir schon vieles angeeignet habe, in allen Facetten verstehenden HA noch nicht.
Gäbe es eine Möglichkeit das ‚irgendwie‘ mit einem Template hinzubekommen?

**Reply by emufan** [reply]
Zeig mal einen Screenshot, was Du wo siehst oder sehen wollen würdest.

**Reply by kwithus** [reply]
![85ECC0E2-4F96-4002-999C-C120DECB400A](https://user-images.githubusercontent.com/20025320/153612384-cbe03786-d348-45be-a8a6-f277b0277d1d.jpeg)

links oben wird für Rollo WZ links im Status ‚offen‘ gezeigt. Im Infofenster sieht man, dass das Rollo aber nicht vollständig geöffnet ist. Es wäre schön, wenn im Status unter dem icon links die entsprechende % Zahl stehen würde.

**Reply by emufan** [reply]
Ah. Du willst da (wirklich? 😉) eine Glance-Card? Da wird es schwierig. Aber ginge mit card_mod auch. state ausblenden und dann stattdessen ein pre: und dann ein Text-Template. 

Ich würde aber so oder so eher auf eine custom:button-card oder paper-buttons-row gehen. Guck beides mal an. Da hast Du es direkter. Oder kannst noch Steuerknöpfe nutzen. Oder Du nutzt Farben.

Denn es wie oben beschrieben. Der Status ist immer offen oder geschlossen. Willst Du was anderes, musst Du Dir was anderes anzeigen als den Status.

**Comment by kwithus** [comment]
ok, erstmal vielen Dank, dass Du Dir die Mühe gemacht hast.
muss mir das mit den custom cards mal ansehen. War mir bis zu kompliziert, bin noch ziemlich neu bei HA und bevorzuge solche cafés, die gänzlich visuell erstellt werden können.
die Glance gibt mir grundsätzlich viel Info auf wenig raum
K-H


Von meinem iPhone gesendet
